format-all.el

```diff
@@ -830,6 +830,12 @@ Consult the existing formatters for examples of BODY."
   (:features)
   (:format (format-all--buffer-easy executable "fmt")))
 
+(define-format-all-formatter csharpier
+  (:executable "dotnet-csharpier")
+  (:install "dotnet install -g csharpier")
+  (:languages "C#")
+  (:features)
+  (:format (format-all--buffer-easy executable "--write-stdout")))
```

Comment:
* Please keep the formatter definitions in alphabetical order.
* Please ensure there is a blank line above and below the formatter definition.

Reply:
:heavy_check_mark:  Moved and formatted formatter definition in formatter package